### DISCUSSION TITLE
Examples in the works!

### DISCUSSION THREAD

**Discussion by klutchdev** [supabase/supabase#727] [Show and tell]
![Screenshot_20210215-180210_Chrome.jpg](https://user-images.githubusercontent.com/58617435/108000542-304dc200-6fb8-11eb-8b2d-e4a418ae1333.jpg)

**Comment by kiwicopple** [comment]
🔥 🔥 🔥 

Dart or React Native?

**Reply by pmauchle** [reply]
Dart

**Reply by kiwicopple** [reply]
Amazing. Let us know when it's done and we can add it to the examples page: https://supabase.io/docs/guides/examples

**Reply by pmauchle** [reply]
I thought it is a question, and I voted for dart 😅

**Reply by kiwicopple** [reply]
haha - I thought you were OP 😅

**Reply by silentworks** [reply]
Lol, this is such a funny exchange.

**Comment by klutchdev** [comment]
🤣🤣🤣
I mean honestly, Dart rules too haha
